src/main/java/se/sundsvall/datawarehousereader/service/mapper/CustomerDetailsMapper.java

```diff
@@ -23,7 +23,10 @@ public static List<CustomerDetails> toCustomerDetails(List<CustomerDetailsEntity
 
 	public static CustomerDetails toCustomerDetails(final CustomerDetailsEntity entity) {
 		return CustomerDetails.create()
+			.withPartyId(entity.getUuid() != null ? entity.getUuid().toLowerCase(): null)
```

Comment:
`Optional.ofNullable(entity.getUuid()).map(String::toLowerCase).orElse(null)`

This will result in exactly the same, but it doesn't create two separate execution paths. I.e. it is easier to get full coverage in jacoco.

Reply:
Nice one, fixed.